ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -212,7 +239,30 @@ public class ButtonTokenSet: ControlTokenSet<ButtonTokenSet.Tokens> {
                     case .large:
                         return theme.typography(.body1Strong)
                     case .medium, .small:
-                        return theme.typography(.caption1Strong)
+                        switch style().isFab {
+                        case true:
+                            return theme.typography(.body2Strong)
```

Comment:
Can you test different text sizes? The FAB styles shouldn't scale, and should instead use the Large Content Viewer.

Reply:
Yep, just update screenshots as well.